test/JDBC/expected/datareader_datawriter-vu-verify.out

```diff
@@ -0,0 +1,631 @@
+-- tsql user=db_roles_l2 password=123
```

Comment:
Let's also add cross-db tests as follows:
1. user is db_writer/db_reader in one db but not in other.
2. user is db_writer in one db and db_reader in other.